packages/maskbook/src/components/InjectedComponents/PayloadReplacer.tsx

```diff
@@ -12,16 +12,23 @@ const useStyle = makeStyles((theme: Theme) => ({
 
 export interface PayloadReplacerProps {
     payload: string
+    theme: Theme
 }
 
-export function PayloadReplacer({ payload }: PayloadReplacerProps) {
+export const PayloadReplacer = withTheme(function ({ payload, theme }: PayloadReplacerProps) {
     const { t } = useI18N()
     const classes = useStyle()
     return (
         <Typography className={classes.text} color="textPrimary" component="span" variant="body1">
-            <Link href={payload.startsWith('https://maskbook.com') ? payload : 'https://maskbook.com'}>
+            <Link
+                ref={(node) => {
+                    if (node) {
+                        node.style.setProperty('color', theme.palette.primary.main, 'important')
```

Comment:
Can you explain why?

Reply:
`<Link style={{color: theme.palette.primary.main + ' !important'}}>` does not work, so use this way.

Reply:
I need to review this later

Reply:
Fine, I'm OK with this as a temporal solution. Please open an issue to address this for fixing it in the future

Reply:
here, you need to add a line of comment. like `// HACK: ...`

Reply:
done